chapters/vulkan_profiles.adoc

```diff
@@ -0,0 +1,481 @@
+
+Developers can create custom profiles to target specific hardware configurations or to define a baseline for their own applications. The process is documented in the link:https://github.com/KhronosGroup/Vulkan-Profiles/blob/main/CUSTOM_PROFILES.md[Creating Custom Profiles] guide. To create a custom profile:
+
+1. Start with an existing profile as a template (see link:https://github.com/KhronosGroup/Vulkan-Profiles/tree/main/profiles[official profiles])
+2. Modify the capabilities to match your requirements
+3. Save the profile as a JSON file
+4. Use the profile in your application using either the header-based or JSON-based approach
+
+You can find examples of custom profiles in the link:https://github.com/KhronosGroup/Vulkan-Samples/tree/main/samples/api/profiles[Vulkan-Samples profiles examples].
+
+Custom profiles are particularly useful for:
+
+* Internal development teams working on multiple applications
+* Defining minimum requirements for specific application categories
+* Creating hardware-specific optimized profiles
+
+Example of creating and using a custom profile:
+
+[source,cpp]
+----
+// Define a custom profile (typically done offline and saved to a JSON file)
+std::string customProfileJson = R"(
+{
+    "file_format_version": "1.0.0",
+    "profiles": {
+        "VP_CUSTOM_GAME_PROFILE": {
+            "version": 1,
+            "api_version": "1.1.0",
+            "label": "Custom Game Profile",
+            "description": "Profile for our game engine",
+            "capabilities": [
+                // ... custom capabilities ...
+            ]
+        }
+    }
+}
+)";
+
+// Load the custom profile
+VkpProfilesFileLoader loader;
+loader.LoadFromString(customProfileJson);
+
+// Use the custom profile
+VkpProfile profile = loader.GetProfile("VP_CUSTOM_GAME_PROFILE");
+// ... use the profile as before ...
+----
+
+== Android Baseline Profile
+
+The Android Baseline Profile is particularly useful for Android developers as it provides a guaranteed set of features that are available on a wide range of Android devices. This profile is documented in the link:https://docs.vulkan.org/spec/latest/appendices/profiles.html#profile-VP_ANDROID_BASELINE_2022[Vulkan Specification] and the link:https://github.com/KhronosGroup/Vulkan-Profiles/blob/main/profiles/VP_ANDROID_BASELINE_2022.json[JSON definition]. This profile includes:
+
+* Core Vulkan 1.1 functionality
+* Essential extensions for Android development
+* Minimum guaranteed limits and features
+
+For Android-specific examples, see the link:https://github.com/KhronosGroup/Vulkan-Samples/tree/main/samples/api/profiles[Vulkan-Samples profiles examples] and the link:https://developer.android.com/ndk/guides/graphics/vulkan-profiles[Android NDK documentation on Vulkan Profiles].
+
+By targeting the Android Baseline Profile, developers can:
+
+* Reduce the need for device-specific code paths
+* Ensure consistent behavior across different Android devices
+* Simplify testing and validation
```

Comment:
Probably a significant use case is that the Android Baseline profiles can be used to filter Vulkan applications in the Android store. If the decice doesn't support the required profile, the application doesn't show up.

Reply:
Yeah, missed opportunity there.  I personally feel that profiles is a good way to suggest Google address fragmentation at this type of level, so this *would be nice.*  Sadly, not the case.